OpenRA.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.27130.2010
+VisualStudioVersion = 15.0.27004.2008

```

Comment:
That looks like a downgrade to me. Can you please unstage it as I merged https://github.com/OpenRA/OpenRA/pull/14523 yesterday? Sorry for the inconvenience.